File: Aula01_Ex06_carro_percent.py
carro = float(input("Insira aqui o preço de fábrica do carro: "))
percentDistribuidor = float(input("Qual o percentual do distribuidor? "))
percentImposto = float(input("Qual o percentual doe imposto? "))

precoDistribuidor = (carro*percentDistribuidor)/100
precoImposto = (carro*percentImposto)/100
custoConsum = carro + precoDistribuidor + precoImposto

print("O preço do distribuidor é R$ {:.2f}, e o do imposto é de R$ {:.2f}. O custo final do carro é de R$ {:.2f}".format(precoDistribuidor, precoImposto, custoConsum))

File: Aula01_Ex06_carro_percent.R
carro<-as.numeric(readline(prompt="Insira o valor do carro: "))
percentDistribuidor<-as.numeric(readline(prompt="Insira o percentual do distribuidor: "))
percentImposto<-as.numeric(readline(prompt="Insira o percentual de Imposto: "))

precoDistribuidor = (carro*percentDistribuidor)/100
precoImposto = (carro*percentImposto)/100
custoConsum = carro + (precoDistribuidor+precoImposto)


print(paste("O preço a ser pago ao distribuidor é de R$", precoDistribuidor, ", e o imposto a ser pago é de R$", precoImposto, ". Seu custo ao consumidor é de", custoConsum))
